### PR TITLE
Weather v1 

### PR DESCRIPTION
**Pull request by Victorgonbu**
# Hi there!

## This weather app information was taken from openWeather  Api. I add the following features for this app:
- Search bar (Were user can search for any location in space-separated city/state/country format)
- Display info about the given location.
- All possible errors for search handled.

>> All features linters fixed.

### CODE REVIEW

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 

Hi @Shaqri your app looks good on the screenshot but I couldn't run it please check the comments below:
::point_down: :point_down: :point_down: :point_down: point_down:

## Required Changes :red_circle: 
 
- [x] Add instructions on how to run the app, and different available scripts.

- [x] Add a live demo link.

- [x] Fix the linter errors
image

- [x] you are missing a necessary config file in your project that's because you added it to .gitignore, I think the config file has an API_secret or something that's not how to hide your API secrets contact me if you want to learn more about it. 
image

- [x] In stall webpack-dev-server package and add this script to your package.json
`"start": "webpack-dev-server --mode development --open"`

- [x] README file contains all that's needed for running the app with the API(s) it consumes. It should have instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them(if necessary).

### Please make all the required changes before you submit for another code review.
## Happy Coding :tophat: 
_Feel free to contact me on slack if you have any questions or need help._
## _Zakariae El Mejdki_

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+
+[Live Demo Link](https://shaqri.github.io/Weather_app/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Git clone https://github.com/Shaqri/Weather_app.git
+- Open the index.html inside the dist folder with your favorite browser
+- Explore the weather from all differents parts of the world you can imagine.
```
Comment (christianotieno) [inline README.md]: ## Required change
- Add additional instructions on how to;
- [x] Install dependencies
- [x] Setup users's  API keys in the environment 
- [x] Run the application.

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️  
Hello @Shaqri 👋 
Congratulations on the hard work done in this project. 👏 
Before these changes get approved let's ensure the project has met all the needed requirements. 
We should always aim for our projects to be `plug & play` as much as possible. See my suggestion below 👇 

### Happy coding 💻 
#### Christian Otieno (CR)

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****


 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**